queries/roc/textobjects.scm

```diff
@@ -0,0 +1,39 @@
+(anon_fun_expr
+  (expr_body) @function.inner) @function.outer
+
+(argument_patterns
+  ((_) @parameter.inner
+    .
+    ","? @parameter.outer) @parameter.outer)
+
+(function_type
+  ((_) @parameter.inner
+    .
+    ","? @parameter.outer) @parameter.outer
+  (#not-eq? @parameter.inner "->"))
+
+(function_call_expr
+  .
+  (_)
+  (parenthesized_expr
+    (expr_body) @parameter.inner) @parameter.outer)
+
+(function_call_expr
+  .
+  (_) @parameter.inner @parameter.outer)
+
+[
+  (annotation_type_def)
+  (alias_type_def)
+  (opaque_type_def)
+] @class.inner @class.outer
+
+(apply_type_arg) @parameter.inner
+
+((#offset! line_comment 0 1 0 -1)) @comment.inner
```

Comment:
This results in an error: try looking [here](https://github.com/nvim-treesitter/nvim-treesitter-textobjects/blob/f5183cea0fda26126e22e789382c208e7b1120f4/queries/v/textobjects.scm#L71) for an example of how this should be done

Reply:
Also, if you happen to be using [lazy](https://github.com/folke/lazy.nvim), you can just put your textobject queries in `~/.local/share/nvim/lazy/nvim-treesitter-textobjects/queries/roc/textobjects.scm` and restart your neovim instance and they should apply locally, if you'd like to be able to do some quick debugging.